example/example.1.0.0.oas.yml

```diff
@@ -44,8 +45,8 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/UnexpectedError'
-        default:
-          $ref: '#/components/responses/UnexpectedError'
+         default:
```

Comment:
random  space? and on the next line.

Reply:
This should be fixed in https://github.com/ukhsa-collaboration/standards-api/commit/ec47653d598d9a80623f559b102949b0001b15e3